tcms/testruns/tests/test_views.py

```diff
@@ -454,6 +456,18 @@ def test_update_selected_case_runs(self):
         self.assertContains(response, _('%d CaseRun(s) updated:') % 1)
         self.assertEqual(self.case_run_1.case.text, "Scenario Version 2")
```

Comment:
This test is wrong! I don't know what I've been thinking back then but this doesn't work like it should. Regardless here's what you have to do:

- Use `case.get_text_with_version(case_text_version)` to grab the actual text (this is how it is displayed in the template) and compare that the text to be displayed is equal to "Scenario Version 2". As it is now you are just resolving the data through the foreign key and this will always pass
- also assert that case_text_version and the latest history_id are equal

If necessary use `self.case_run_1.refresh_from_db()` to refresh the values after save.